### DISCUSSION TITLE
Iconos

### DISCUSSION THREAD

**Discussion by Ballwictb** [IsraC0d33/Proyecto1Marcas#7] [Ideas]
Ideas para encontrar una api de iconos coloridos.

@MuiGt  @IsraC0d33 @MarcosAcedoLlerena @MarioGalanEscudero

**Comment by IsraC0d33** [comment]
a que te refieres en concreto??

**Reply by Ballwictb** [reply]
Me refiero a una api o documento que tenga ya iconos sobre redes sociales y que tengan color.

Ya que seria totalmente contraproducente crearlos nosotros.

**Reply by MuiGt** [reply]
No entiendo, quieres buscar logos de redes sociales en svg para ponerlos?

**Reply by Ballwictb** [reply]
No sino una Api o documento (P.ej: CSS) que ya tenga los logos creados.

**Comment by MarcosAcedoLlerena** [comment]
VALE GRACIAS
